cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -953,6 +953,19 @@ func getEnvWithDefaultString(k string, defaultVal string) string {
 	return v
 }
 
+func getEnvWithDefaultDuration(k string, defaultVal time.Duration) time.Duration {
```

Comment:
There are a few other durations like the sync interval, vacuum interval, and merge interval. I can use this for those arguments as well unless anyone knows of a reason not to